pydoctor/themes/base/nav.html

```diff
@@ -1,28 +1,99 @@
 
 <nav class="navbar navbar-default" xmlns:t="http://twistedmatrix.com/ns/twisted.web.template/0.1">
   <meta name="pydoctor-template-version" content="1" />
-  <div class="container" >
+  <div class="container">
 
     <div class="navbar-header">
       
       <div class="navlinks">
-          <span class="navbar-brand">
-            <t:slot name="project" /> <a href="index.html">API Documentation</a>
-          </span>
+        <span class="navbar-brand">
+          <t:slot name="project" /> <a href="index.html">API Documentation</a>
+        </span>
 
-          <a href="moduleIndex.html">
-            Modules
-          </a>
+        <a href="moduleIndex.html">
+          Modules
+        </a>
 
-          <a href="classIndex.html">
-            Classes
-          </a>
+        <a href="classIndex.html">
+          Classes
+        </a>
 
-          <a href="nameIndex.html">
-            Names
-          </a>
+        <a href="nameIndex.html">
+          Names
+        </a>
+
+        <div id="search-box-container">
+          <div class="input-group">
+            <input id="search-box" type="text" name="search-query" placeholder="Search..." aria-label="Search" minlength="2" class="form-control"/>
```

Comment:
We definitely want to set `autocomplete=off` because otherwise browsers may show their own autocompletion on top of the JS autocompletion.

Reply:
I think `type=search` disables that.

Reply:
No it does not ... at least not in Firefox, I still get the Firefox autocompletion.

Reply:
Is it very bad? Because right now, the ways it works it's not really autocompletion... For instance, if you type "pydoctor.tes" there will be no results... So maybe it makes sens to keep it that way?

But it will show up if we search for "pydoctor.tes*". So maybe there is something to do ?

Let's note that "pydoctor.test" gives one result whereas "pydoctor.test*" gives 592 results

Reply:
Yes it is very bad. It suggests searches I have performed on completely different sites.

Reply:
What do you mean? I don’t get it

Reply:
Hum, I get your point